pkg/vault/test/VaultAuthorization.test.ts

```diff
@@ -243,10 +245,11 @@ describe('VaultAuthorization', function () {
     const BUFFER_PERIOD_DURATION = MONTH;
 
     sharedBeforeEach(async () => {
-      authorizer = await deploy('TimelockAuthorizer', { args: [admin.address, ZERO_ADDRESS, MONTH] });
-      vault = await deploy('Vault', {
-        args: [authorizer.address, ZERO_ADDRESS, PAUSE_WINDOW_DURATION, BUFFER_PERIOD_DURATION],
```

Comment:
Question: what was this `ZERO_ADDRESS` that is now gone?

Reply:
The Vault